dash/_callback.py

```diff
@@ -272,8 +279,8 @@ def insert_callback(
     return callback_id
 
 
-# pylint: disable=R0912, R0915
-def register_callback(  # pylint: disable=R0914
+# pylint: disable=too-many-branches,too-many-statements
```

Comment:
Same question as above -- can this function be lightened a bit by moving some logic to sub-functions?

Reply:
@T4rk1n any thoughts on ^ ? Not to be a stickler and it's not a blocker for merging this PR but would love to see this function simplified if possible rather than disabling the linter

Reply:
I didn't disable anything in this PR, just changed to the rulename instead of number. The settings for these rules is kinda low imo,
- too-many-branches=15 is kinda high, but this function is wrapped multiple times in decorators which is why it's triggered here.
- too-many-statements=50 is kinda low for a project this size and keep getting triggered everywhere, we also have functions within functions which actually separate the code inside but the linter isn't too smart here and the top function accumulates all the statements.

Reply:
Oh I see now, thanks for the explanation. 👍